common/djangoapps/course_modes/helpers.py

```diff
@@ -83,3 +87,18 @@ def _enrollment_mode_display(enrollment_mode, verification_status, course_id):
         display_mode = enrollment_mode
 
     return display_mode
+
+
+def get_course_final_price(user, sku, course_price):
+    """
+    Return the course's discounted price for a user if user is eligible for any otherwise return course original price.
+    """
+    price_details = {}
+    try:
+        price_details = ecommerce_api_client(user).baskets.calculate.get(
+            sku=[sku],
+            username=user.username,
+        )
+    except (SlumberBaseException, ConnectionError, Timeout) as exc:     # pylint: disable=unused-variable
```

Comment:
Seems strange to import these exceptions just to ignore them. Could just catch a general exception, but I'm not a fan of that either.

Reply:
I am more of a specific exception guy. In this specific case, we don't need to do anything in case of an exception.